README.md

```diff
@@ -68,6 +68,8 @@ Use the init command to initialize Dapr. On init, multiple default configuration
 dapr init
 ```
 
+> For Linux users, if you run your docker cmds with sudo or the install path is the default `/usr/local/bin`, you need to use "**sudo dapr init**"
```

Comment:
can it say `/usr/local/bin` (default install path)

Reply:
it flows better now reading as a whole.